.mergify.yml

```
@@ -63,7 +63,7 @@ merge_protections:
   - name: Enforce conventional commit
     description: Make sure that we follow https://www.conventionalcommits.org/en/v1.0.0/
     if:
-      - base = main
+      - "-draft"
```

We use draft our self and we should have this check on our PR.
```suggestion
      - "-base~=^mergify/merge-queue/"
```

I'm unsure about the `/` excaping.

---

No sure it's necessary since MP won't run on MQ PRs.

---

Maybe we should just remove the `if`, WDYT?

---

Works for me.